tests/browser/browser_test.js

```diff
@@ -13,12 +14,29 @@ XPCOMUtils.defineLazyModuleGetters(this, {
 // The path of the add-on file relative to `getTestFilePath`.
 const ADDON_PATH = "quicksuggest.xpi";
 const ABOUT_BLANK = "about:blank";
+const URLBAR_PROVIDER_NAME = "ProviderDynamicPalmTree";
```

Comment:
I noticed that when I renamed PalmTree -> QuickSuggest I missed that one, could be a good time to clean up the last of that rename

Reply:
Good point, I'll fix that.